lib/Data/XlsCell.hs

```diff
@@ -0,0 +1,35 @@
+-- |
+-- Module      : Data.XlsCell
+-- Copyright   : (c) 2022 Olaf.Klinke
+--
+-- License     : BSD-style
+-- Maintainer  : olaf.klinke@phymetric.de
+-- Stability   : experimental
+-- Portability : GHC
+--
+-- Static Excel cell values
+--
+{-# LANGUAGE DeriveFunctor, FlexibleInstances #-}
+module Data.XlsCell (CellF(..),Cell,cellToString) where
+import Data.String (IsString(..))
+import Text.Printf (printf)
+
+-- | extensible 'Cell' type
+data CellF o = NumericalCell Double
+    | TextCell String
+    | BoolCell Bool 
+    | OtherCell o
+    deriving (Functor,Show,Eq)
```

Comment:
Is there a way/possiblity for users to use this type? otherwise we should not expose this type and if that's the case then we can just use a `Cell` type instead of using it as a synonym to `CellF ()`. Will keep things simple for users.